### DISCUSSION TITLE
Why are DEB updates considered system updates?

### DISCUSSION THREAD

**Discussion by eskatos** [hydraulic-software/conveyor#139] [Q&A] [answered]
When an update to the DEB package for my JVM Compose app is available, they are considered as "System Updates" by Ubuntu/Software app.

![image](https://github.com/user-attachments/assets/7142cdb2-a5f7-4f92-8f1a-34a3589f32f6)

Installing "System Updates" requires two reboots.

Is there a reason for this?
Is it possible to make the DEB updates not considered as "System Updates" and requiring two reboots?

**Comment by mikehearn** [comment, marked as answer]
There's nothing in the DEB itself that marks it as such, actually DEB as a format doesn't have any concept of a "system update" at all. So this must be how Ubuntu treats any software that isn't shipped using Snap, which is their attempt to replace the format entirely. Unfortunately building snaps isn't currently on the roadmap. I don't think many Linux users manage their software via the built-in store app anyway.

**Reply by eskatos** [reply]
Thanks. I read a bit about how Ubuntu is doing this. This doesn't happen if you simply `apt upgrade` or use any other package manager :shrug:
